corehq/apps/campaign/models.py

```diff
@@ -86,6 +86,10 @@ class WidgetType:
         REPORT: DashboardReportForm,
     }
 
+    @classmethod
+    def get_form_class(cls, widget_type):
+        return cls.FORM_CLASS.get(widget_type)
```

Comment:
You would never expect `widget_type` not to be a known widget type, so fail loudly.
```suggestion
        return cls.FORM_CLASS[widget_type]
```